src/components/SearchComponent.vue

```diff
@@ -14,12 +14,19 @@ defineProps({
   searchLabel: {
     type: String,
     required: true
+  },
+  placeholder: {
+    type: String,
+    default: ''
+  },
+  id: {
+    type: String,
+    default: 'select-search'
```

Comment:
Forcer un `id` pour éviter les doublons en cas d'utilisation multiple.

```suggestion
    required: true
```